### PR TITLE
S1 us8 accueil

### PR DESCRIPTION
**Pull request by Vannou28**

### CODE REVIEW

**Review by khadimb — DISMISSED** [review]
revue ok

File: public/assets/css/indexHome.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    height: var(--heightSection);
+    background-image: var(--backgroundImageHome);
+    display: flex;
+    justify-content: space-around;
+    text-align: center;
+}
+
+.home img{
+    height: auto;
+    padding:  30px;
```
Comment (LucyMerlier) [inline public/assets/css/indexHome.css:26]: Avoid magic values

File: public/assets/css/indexHome.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+.h2a{
+    padding:  30px;
```
Comment (LucyMerlier) [inline public/assets/css/indexHome.css:30]: avoid magic values

File: public/assets/css/indexHome.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    justify-content: space-around;
+    text-align: center;
+}
+
+.home img{
+    height: auto;
+    padding:  30px;
+}
+
+.h2a{
```
Comment (LucyMerlier) [inline public/assets/css/indexHome.css:29]: misclick?

File: public/assets/css/indexHome.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    padding:  30px;
+    align-self: center;
+}
+
+.home p{
+    font-size: var(--p-font-size);
+    text-shadow: 2px 2px rebeccapurple;
```
Comment (LucyMerlier) [inline public/assets/css/indexHome.css:36]: avoid magic values

File: public/assets/css/indexHome.css [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+.home a{
+    margin: 20px;
```
Comment (LucyMerlier) [inline public/assets/css/indexHome.css:40]: same as above

File: public/assets/css/indexHome.css [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+}
+@media screen and (max-width: 700px) {
+
+    .home{
+        flex-direction: column;
+        height: auto;
+    }
+
+    .home img{
+        height: 300px;
```
Comment (LucyMerlier) [inline public/assets/css/indexHome.css:61]: idem

File: src/View/Home/index.html.twig [hunk 7]
```diff
@@ -1,5 +1,14 @@
 {% extends 'layout.html.twig' %}
+{% block stylesheet %}
+    <link rel="stylesheet" href="/assets/css/indexHome.css">
+{% endblock %}
 
 {% block content %}
-    <h1>Nathalie Milliet</h1>
+    <section class="home">
+        <img src="../../assets/images/home/LOGO.png" class="img-fluid" alt="Responsive image">
```
Comment (LucyMerlier) [inline src/View/Home/index.html.twig:8]: better use absolute paths

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
avoid magic values, also I didn't check them all but are you really using all those images in this single US? oh and finally : be coherent when naming your branches please :)

File: src/View/Home/index.html.twig [hunk 8]
```diff
@@ -1,5 +1,14 @@
 {% extends 'layout.html.twig' %}
+{% block stylesheet %}
+    <link rel="stylesheet" href="/assets/css/indexHome.css">
+{% endblock %}
 
 {% block content %}
-    <h1>Nathalie Milliet</h1>
```
Comment (RaphaelBS-WCS) [inline src/View/Home/index.html.twig:4]: All that must be on your own view sheet

**Review by RaphaelBS-WCS — COMMENTED** [review]
I left you some comments

File: src/View/Home/index.html.twig [hunk 9]
```diff
@@ -1,5 +1,8 @@
 {% extends 'layout.html.twig' %}
+{% block stylesheet %}
+    <link rel="stylesheet" href="/assets/css/indexHome.css">
```
Comment (RaphaelBS-WCS) [inline src/View/Home/index.html.twig:3]: it seems to me that it have to be on your home.html.twig instead of here...

**Review by RaphaelBS-WCS — COMMENTED** [review]
a last thing...

File: src/View/Home/index.html.twig (hunk 9 above)
Comment (RaphaelBS-WCS) [reply src/View/Home/index.html.twig]: we see together that you have corrected this.

**Review by RaphaelBS-WCS — DISMISSED** [review]
I approve!

File: public/assets/css/indexHome.css [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+    .home p{
+        font-size: calc(var(--font-big-size)/1.5);
+    }
+}
+
+/*--------------------------------------------------  768px   ------------------*/
+@media screen and (max-width: 768px) {
+
+    .home{
+        flex-direction: column;
```
Comment (LucyMerlier) [inline public/assets/css/indexHome.css:48]: could use bootstrap to avoid some media queries

File: public/assets/css/style.css [hunk 11]
```diff
@@ -1,8 +1,43 @@
+
+
 @import url('https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap%27');
 
 :root {
     --primary-color : #4A235A ;
     --secondary-color: #8E44AD;
     --background-color : #FEF8FF;
+    --placeholder : #666;
```
Comment (LucyMerlier) [inline public/assets/css/style.css:9]: :smiling_imp: 
what's that for though?

File: src/View/Home/home.html.twig [hunk 12]
```diff
@@ -0,0 +1,13 @@
...
+
+{% block content %}
+    <section class="home">
+        <img src="assets/images/home/LOGO.png" class="img-fluid" alt="Responsive image">
+        <div class="contain-p-a">
+            <p>Découvrez ce que l'avenir vous réserve</p>
+            <a href="#contact" type="button" class="btn colorButton btn-lg ">Me contacter</a>
+        </div>
+    </section>
+{% endblock %}
```
Comment (LucyMerlier) [inline src/View/Home/home.html.twig:13]: components should not contain twig blocks, and the name of the file should start with an underscore

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments. also, incoherent branch naming

**Review by RaphaelBS-WCS — DISMISSED** [review]
Approved

File: src/View/Home/_home.html.twig [hunk 13]
```diff
@@ -0,0 +1,14 @@
+
+<section class="home" id="home">
+    <div class="container">
+        <div class="row d-flex align-items-center justify-content-center">
+            <div class="col-12 col-sm-12 col-md-6 col-lg-5">
+                <img src="assets/images/home/LOGO.png" class="img-fluid" alt="Responsive image">
```
Comment (LucyMerlier) [inline src/View/Home/_home.html.twig:6]: not a valid alt attribute

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
do you really need all those images for this US? + some of them weigh more than 200KB

**Review by RaphaelBS-WCS — APPROVED** [review]
I approve

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
purple_nebula.jpg is quite heavy -> possible to make it lighter?

File: public/assets/css/indexHome.css [hunk 14]
```diff
@@ -0,0 +1,25 @@
+@import 'style.css';
+
+.home{
+
```
Comment (RaphaelBS-WCS) [inline public/assets/css/indexHome.css:4]: extra line 4

**Review by RaphaelBS-WCS — APPROVED** [review]
It's good to me!

**Review by RaphaelBS-WCS — DISMISSED** [review]
I approve

**Review by LucyMerlier — DISMISSED** [review]

**Review by RaphaelBS-WCS — APPROVED** [review]
i approved

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**